Cargo.lock

```diff
@@ -303,7 +303,7 @@ checksum = "e78d4f1cc4ae33bbfc157ed5d5a5ef3bc29227303d595861deb238fcec4e9457"
 
 [[package]]
 name = "enclave_build"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Why do we bump the version here? Is the version 0.1.0 released on crates.io?

Reply:
Since it's a significant change I thought it would make sense to change the version here, but I don't know how this library's maintainers manage versioning.

Reply:
If it's not on crates.io, I don't see a point in updating the version here.

Reply:
I have no idea, I think it's up to the maintainers to choose here.